.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: ['3.7', '3.8', '3.9', '3.10']
```

Comment:
As mentioned in the ticket, no need to drop by 3.6 in order to depend on psycopg 3.0.x